scripts/nexus-export.sh

```diff
@@ -54,8 +55,31 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 1000Gi
+      storage: $backupSpaceNeeded
 EOF
+elif [[ ${backupSpaceNeeded} != $(kubectl get pvc -n nexus nexus-bak -o jsonpath='{.status.capacity.storage}') ]]; then
+  echo "The backup PVC size does not match the nexus-data PVC."
+  read -p "Would you like to delete the nexus-bak PVC and recreate it to match the size? (y/n)" -n 1 delete
+  delete=${delete,,}
+  if [ "$delete" = "y" ]; then
+    kubectl -n nexus delete pvc nexus-bak
+    cat << EOF | kubectl -n nexus create -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nexus-bak
+spec:
+  storageClassName: k8s-block-replicated
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: $backupSpaceNeeded
+EOF
+  else
```

Comment:
Where is the n case? this just checks that the user said yes, if they say no shouldn't we let them fail?

I'd see this as yes/no/abort and check for yes/no cases with the latter being the fall through.

Reply:
In my eyes its a yes or abort case. With no its going to fail silently creating the tar. Saying No here means they dont want to make a new backup. Maybe a better phrasing for the question is needed.

Reply:
oki doki works for me was just wondering if we wanted to let them fail in the no case or not.